js_modules/dagster-ui/packages/ui-core/src/app/AppTopNav/activePathMatchers.tsx

```diff
@@ -17,7 +24,10 @@ export const settingsPathMatcher: MatcherFn = (_, currentLocation) => {
   const {pathname} = currentLocation;
   return (
     pathname.startsWith('/settings') ||
-    (pathname.startsWith('/locations') && !pathname.includes('/asset-groups/'))
+    (pathname.startsWith('/locations') &&
+      !pathname.includes('/asset-groups/') &&
+      !automationPathMatcher(_, currentLocation) &&
+      !jobsPathMatcher(_, currentLocation))
```

Comment:
I am not 100% sure, but I think these last two are meant to be outside the closing paren and only 27+28 should be `&&` together?

Reply:
Hmm, I think it's correct as it is -- we want to filter out any of the `/locations` paths that should instead be "Automation" or "Jobs" paths.